MusicHub/tracks/track_service.py

```diff
@@ -0,0 +1,3 @@
+def get_filename_from_track(request):
+    request.data["filename"] = request.data.get("track").name
+    return request.data
```

Comment:
pretty controversial. the naming is `get_` but actually mutates input ... let's discuss